charts/mod-dcb/values.yaml

```diff
@@ -0,0 +1,138 @@
+# Default values for mod-circulation.
```

Comment:
Please fix name in comment for all newely added charts

Reply:
Done